Joedmin-hw3/Joedmin-hw3/DuplicateUserCredentialsException.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
+
+namespace Joedmin_hw3
```

Comment:
Namespace names should be CapsCase.  Also `Snake_case` with a leading capital letter is not used anywhere.

Reply:
I have it named like this just case my folder, but won't name it like this anymore